src/elm/Main.elm

```diff
@@ -1332,6 +1346,24 @@ refreshPage model _ =
             Cmd.none
 
 
+{-| refreshPageHidden : refreshes Vela data based on current page and build status when tab is not visible
+-}
+refreshPageHidden : Model -> RefreshData -> Cmd Msg
+refreshPageHidden model _ =
+    let
+        page =
+            model.page
+    in
+    case page of
+        Pages.Build org repo buildNumber _ ->
+            Cmd.batch
+                [ refreshBuild model org repo buildNumber
```

Comment:
when the page is hidden, we only need build status for the favicon.
this is to limit the load put on the server, while getting the functionality we need across inactive tabs.